tests/contrib/rename_image_test.sh

```diff
@@ -0,0 +1,26 @@
+# Copyright 2015 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+
+set -ex
+BASEDIR=$(dirname "$0")
+
+if [ $(./contrib/extract_image_name.sh ./tests/contrib/renamed_image.tar) = 'new_image_repo:new_image_tag' ]
```

Comment:
all these values should be passed as args in the BUILD rule so that they can be moved / renamed w/o having to change this script

Reply:
Also, if this is the only place where you're using extract_image_name.sh, just do the extracting of the name directly here (no need to separate extracting from the evaluating if its the same as the one produced by the rules)

Reply:
I thought this might be useful in the future, so I made it a separate script. That's also why I put it in contrib. Should I just do it in the this file and remove the script?

Reply:
Fixed.